package.json

```
@@ -62,8 +62,8 @@
     "serverless-finch": "^2.6.0",
     "serverless-stack-output": "^0.2.3",
     "strip-color": "^0.1.0",
-    "twilio": "^3.54.1",
-    "twilio-video": "^2.10.0",
+    "twilio": "^3.39.3",
+    "twilio-video": "^2.11.0",
```

Can we move to 2.12?
```suggestion
    "twilio-video": "^2.12.0",
```